### DISCUSSION TITLE
How was the `host.internal` done?  It looks very useful.

### DISCUSSION THREAD

**Discussion by mendhak** [sakai135/wsl-vpnkit#81] [General]
I've just been looking at this tool and the README mentions that the Windows host is accessible using `host.internal`.  That feels really useful compared to the [options listed on the WSL networking page](https://docs.microsoft.com/en-us/windows/wsl/networking).  

This could make for consistent development, eg I could put something in a setting file like `some_api_url = http://host.internal:1234` instead of extracting an IP address.  

I'm wondering though, how was this done?  Is host.internal part of VPNKit.exe itself? 

Thanks for this useful script.

**Comment by sakai135** [comment]
@mendhak Yes, the `host.internal` is done by VPNKit. When you run `vpnkit.exe --help`, one of the options is `--host-names` which defaults to `host.internal`.
